guides/source/action_view_overview.md

```diff
@@ -318,6 +318,28 @@ You can also specify a second partial to be rendered between instances of the ma
 
 Rails will render the `_product_ruler` partial (with no data passed to it) between each pair of `_product` partials.
 
+#### Explicit Locals
```

Comment:
@byroot thinking about this some more, what do you think about calling this `Strict Locals` instead of `Explicit Locals`? The latter feels better to me.

Reply:
👍

Reply:
https://github.com/rails/rails/pull/45727